src/vmm/src/lib.rs

```diff
@@ -317,6 +321,10 @@ pub struct Vmm {
     vcpus_handles: Vec<VcpuHandle>,
     // Used by Vcpus and devices to initiate teardown; Vmm should never write here.
     vcpus_exit_evt: EventFd,
+    // seccomp_filters are only needed in VMM for hotplugging vCPUS.
+    // Since this is something that can only occur on x86_64, we need to allow
+    // dead code for aarch64.
+    #[allow(dead_code)]
```

Comment:
Does it make sense to instead use `[cfg(target_arch = "x86_64")]`? I think I would prefer it, unless it pollutes the code with too many conditional compilation flags.